app/pages/changeActivity/ContractPackingPage.scala

```diff
@@ -29,4 +30,15 @@ case object ContractPackingPage extends QuestionPage[Boolean] {
   override def toString: String = "contractPacking"
 
   override val url: Mode => String = mode => routes.ContractPackingController.onPageLoad(mode).url
+
+  override val redirectConditions: UserAnswers => List[RequiredPage] = userAnswers => {
+    def isSmallOrLargeProducer(userAnswers: UserAnswers): Boolean =
+      userAnswers.get(AmountProducedPage).contains(AmountProduced.Large) || userAnswers.get(AmountProducedPage).contains(AmountProduced.Small)
+
+    List(
+      RequiredPage(AmountProducedPage),
+      RequiredPage(ThirdPartyPackagersPage, additionalPreconditions = List(userAnswers.get(AmountProducedPage).contains(AmountProduced.Small))),
+      RequiredPage(OperatePackagingSiteOwnBrandsPage, additionalPreconditions = List(isSmallOrLargeProducer(userAnswers)))
+    )
+  }
```

Comment:
What if Amount Produced has not been answered?  i.e. I get to the amount produced page and manually navigate here?

Reply:
Line 39?